app/src/main/res/layout/tab_fragment_main.xml

```diff
@@ -278,7 +278,7 @@
                 android:layout_height="@dimen/buttonContinueHeight"
                 android:layout_below="@+id/orderDisplay"
                 android:layout_centerInParent="true"
-                android:background="@drawable/ic_continue_button"
+                android:background="@color/buttonContinueColor"
```

Comment:
why don't you use the picture and not just the color?

Reply:
changed it